app/views/partners/dashboards/_requests_in_progress.html.erb

```diff
@@ -21,11 +21,15 @@
 
           <td class="p-4"><%= request.total_items %></td>
           <td class="p-4 d-flex flex-wrap">
-            <% request_item = request.request_items.map { |json| RequestItem.from_json(json, request, @inventory) } %>
-
-            <% request_item.map do |item| %>
+            <% request.item_requests.each do |item_request| %>
               <span class="p-1 mr-1 mb-2 lg:mb-0 border border-dark rounded-1">
-                <%= item.quantity %> <%= item.name %>
+                <% if Flipper.enabled?(:enable_packs) && item_request.request_unit %>
+                  <%= pluralize(item_request.quantity, item_request.request_unit) %>
+                  --
```

Comment:
Can we use an emdash instead of double-hyphens? (I don't remember if we have to use the html entity `&mdash;` or can just use `—`. If the latter, here's the suggestion)
```suggestion
                  —
```

Reply:
Envy em (N.V.M.)…I see CL specified double-hyphens. That's fine if you wanna keep that.